src/bitcoin-chainstate.cpp

```diff
@@ -28,7 +28,7 @@
 #include <functional>
 #include <iosfwd>
 
-const std::function<std::string(const char*)> G_TRANSLATION_FUN = nullptr;
+__attribute__ ((visibility("default"))) const std::function<std::string(const char*)> G_TRANSLATION_FUN = nullptr;
```

Comment:
In commit "b-cs: Explicitly mark G_TRANSLATION_FUN as visible" (61ab5fa1d0a8d0e77b49a24b1b7035deaea9b7c8)

Does the fact that this commit fixes a link error imply that previous commits will not build? If so, this seems like something that we might want to fix so the previous commits are functional.

In any case, it would be good to get rid of this line entirely at some point. The application shouldn't need to export its own symbols to use the library. The translation global just be a global in the library if it has to be a global.

Reply:
Done a36c0ff4af800e5499bbe7aca3137581251e71b3